AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/deployments/DeploymentsRest.java

```diff
@@ -363,6 +367,32 @@ public Response getAllDeploymentParameterKeys() {
         return Response.status(Status.OK).entity(deploymentParameters).build();
     }
 
+    @POST
+    @Path("/deploymentParameterKeys/")
+    @ApiOperation(value = "Adds one parameter key")
+    public Response addOneParameterKey(DeploymentParameterDTO deploymentParameterDTO) {
+        Key keyToBeAdded = new Key(deploymentParameterDTO.getKey());
+        keyRepository.createDeployParameterKey(keyToBeAdded);
+        return Response
+                .status(CREATED)
+                .entity(keyToBeAdded)
+                .location(URI.create("/deploymentParameterKeys/" + keyToBeAdded.getName()))
+                .build();
+    }
+
+    @DELETE
+    @Path("/deploymentParameterKeys/{keyName}")
+    @ApiOperation(value = "Deletes one parameter key")
+    public Response deleteOneKey(@PathParam("keyName") String keyName) {
+        Key keyToBeRemoved = keyRepository.findFirstKeyByName(keyName);
+        if (keyToBeRemoved != null) {
+            deploymentParameterBoundary.deleteDeployParameterKey(keyToBeRemoved);
+            return Response.ok().build();
+        } else {
+            return Response.status(Response.Status.NOT_FOUND).build();
```

Comment:
throw NotFoundException instead of  if-else. So the endpoint only returns ok-responses. The error will be mapped in the service by using `NotFoundExceptionMapper`